oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -18,14 +18,10 @@
  */
 package oshi.software.os;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import org.junit.Test;
-
 import oshi.SystemInfo;
-import oshi.software.common.AbstractProcess;
+
+import static org.junit.Assert.*;

```

Comment:
FYI, my source formatter/IDE will put this back the way it was (individual imports) whenever I eventually edit this.